sphinx-docs/usermanual/userman_coach.rst

```diff
@@ -28,13 +28,6 @@ Overview
 
 The overview will display the activity of the group: the total hours logged, the average progress in exercises, the total exercises attempted, and the total time spent on content.
 
-.. screenshot::
```

Comment:
Actually, these might also have been failing because of the window size. Let me try it again before I nix them.